app/helpers/language_helper.rb

```diff
@@ -7,6 +7,10 @@ def rtl?
     %w(ar he).include?(Globalize.locale.to_s)
   end
 
+  def rtl_lang?(lang)
```

Comment:
Could we tweak this slightly, both for clarity and so there's a little less repetition in the view? 

```
def rtl_language?(language)
  %w(ar he).include?(language.short)
end
```

That way, the views can do `rtl_language?(admin_post.language)`